src/libraries/Directory.Build.targets

```diff
@@ -196,5 +199,9 @@
       </ItemGroup>
     </When>
   </Choose>
-
+  
+  <PropertyGroup Condition="'$(ExcludeCurrentFrameworks)' == 'true'">
+    <ExcludeFromPackage Condition="'$(TargetFramework)' == 'netcoreapp5.0'">true</ExcludeFromPackage>
+    <ExcludeFromPackage Condition="'$(TargetFramework)' == 'net472'">true</ExcludeFromPackage>
```

Comment:
We don't publish any packages with net472 in it?

Reply:
Only Packages that set ```ExcludeCurrentFrameworks``` will exclude net472. This property is not true for all packages

Reply:
But couldn't there be packages where we don't want the latest netcoreapp assets in them but net472 assets? I don't really understand why net472 is part of this current list.

Reply:
Also shouldn't this use the latest tfm properties instead of hardcoding there values here?

Reply:
> But couldn't there be packages where we don't want the latest netcoreapp assets in them but net472 assets? I don't really understand why net472 is part of this current list

Currently there is no such case. If such a case arises we can define two properties instead of one or directly use ExcludeFromPackage  property in the csproj.

> Also shouldn't this use the latest tfm properties instead of hardcoding there values here?

Yes this can be done. I missed that as I was playing with the exact position where we should place these.

Reply:
I agree with @ViktorHofer -- I do see that we might end-up including `net472` in packages but not `CurrentNetCoreApp`. I would instead add two separate properties one for each tfm. Maybe even an `ItemGroup`.

Reply:
Can you explain what a "current" target framework is and why net472 is part of that list as there's already net48 released?

Reply:
> why net472 is part of that list as there's already net48 released?

We haven't retargeted to net48, so CurrentNetFramework points to net472.

Reply:
Of course, but I'm asking about the semantics here. There are two issues with this:

1. The name is misleading, current isn't true for .NET Framework.
2. I don't understand why we would ever exclude both .NET Core and .NET Framework latest versions from a package. What is the use-case here?

Reply:
> The name is misleading, current isn't true for .NET Framework.

we use $(NetFrameworkCurrent) for net 472, I was being consistent with that.

> I don't understand why we would ever exclude both .NET Core and .NET Framework latest versions from a package. What is the use-case here?

I don't know why but we have bunch of projects where we exclude net472 and netcoreapp5.0 from package like System.Data.Odbc, Sysem.Diagnostics.DiagnosticsSource, System.IO.FileSystem.AccessControl

Reply:
>  don't understand why we would ever exclude both .NET Core and .NET Framework latest versions from a package. What is the use-case here?

For the purposes of Source-build and testing we must build something against the latest framework.  

We don't want to include that in the package most times because doing so would bloat the packages or create a moving target.  ASP.NET-extensions is doing this and it is actually creating unintentional breaking changes when the re-version their repo.  Packages which shipped a netcoreapp3.0 asset switched to netcoreapp3.1 in 3.1 release, if you  install the 3.1 version of the package in a 3.0 app it no longer gets the netcoreapp build, and regresses to the netstandard implementation which is broken.

> shouldn't this use the latest tfm properties

+💯 if we use the actual values this is going to get lost in v-next.

Reply:
> For the purposes of Source-build and testing we must build something against the latest framework.

How does .NET Framework fit into that picture? I thought we only care about building netcoreapp for sourcebuild.